src/pudl/analysis/state_demand.py

```diff
@@ -266,15 +265,18 @@ def load_ventyx_hourly_state_demand(path: str) -> pd.DataFrame:
 # --- Datasets: FERC 714 hourly demand --- #
 
 
-def load_ferc714_hourly_demand_matrix(
-    pudl_out: pudl.output.pudltabl.PudlTabl,
+@multi_asset(
+    compute_kind="Python",
+    outs={
+        "raw_hourly_demand_matrix_ferc714": AssetOut(),
```

Comment:
Might be more optimal to make `utc_offset_ferc714` an asset that takes in `raw_hourly_demand_matrix_ferc714`, rather than use a multi-asset. Thoughts?